packages/cli/src/ui/themes/shades-of-purple.ts

```
@@ -0,0 +1,348 @@
+    },
+    'hljs.hljs-line-numbers .hljs-ln-numbers': {
+      color: shadesOfPurpleColors.Gray,
+      paddingRight: '1em',
+    },
+    'hljs.hljs-line-numbers .hljs-ln-code': {
+      paddingLeft: '1em',
+    },
+
+    // Selection styling
+    'hljs::selection': {
+      background: shadesOfPurpleColors.AccentBlue + '40', // 40 = 25% opacity
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)